client/src/components/Sidebar/SidebarBody/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { SIDEBAR_ITEM } from '../'
+import UserList from '../UserList'
+import ChatHistory from '../ChatHistory'
+
+const SidebarBody = ({ currentItem }: { currentItem: SIDEBAR_ITEM }) => {
+  return (
+    <div className="sidebar--body">
+      {currentItem === SIDEBAR_ITEM.USERS && <UserList />}
```

Comment:
nbd but I think it'd be more readable to have a boolean variable for currentItem === SIDEBAR_ITEM.USERS